### ISSUE TITLE
opening multiple folders via "enter" is unintuitive

### ISSUE THREAD

**Issue by hasufell** [IgnorantGuru/spacefm#653]
given the following folders are selected:
```
/tmp/foo
/tmp/bar
/tmp/baz
```
If you hit enter, you will get:
```
tab1 -> foo
tab2 -> baz
tab3 -> foo
```

I think `enter` here should behave in the same way as `Open -> In Tab -> New`.

**Comment by RaveYoda** [comment]
I agree with the logic presented. If i'm opening new tabs via enter it doesn't mean that I want the tab i'm currently in to change to the location that's first in the list. Doung a new seperate tab for all selected items is cleaner.
